accumulo-store/src/main/java/gaffer/accumulostore/key/IteratorSettingFactory.java

```diff
@@ -55,7 +55,20 @@
      * @return A new {@link IteratorSetting} for an Iterator capable of filtering {@link gaffer.data.element.Element}s based on a {@link View}
      * @throws IteratorSettingException if an iterator setting could not be created
      */
-    IteratorSetting getElementFilterIteratorSetting(final View view, final AccumuloStore store)
+    IteratorSetting getElementPreAggregationFilterIteratorSetting(final View view, final AccumuloStore store)
+            throws IteratorSettingException;
+
+    /**
```

Comment:
This javadoc and the one above should differentiate between pre and post aggregation.